example/spec/parallel/orca/OrcaBackupAndRecoverySpec.js

```diff
@@ -162,11 +163,12 @@ describe('The S3 Ingest Granules workflow', () => {
       }
       const recoveryWorkflowName = get(collectionsApiResponse, 'meta.granuleRecoveryWorkflow');
 
+      collectionId = encodedConstructCollectionId(collection.name, collection.version);
```

Comment:
This method was introduced to URL encode the collectionId. If we're using the collectionId in a body, not a path, I think we could probably use `constructCollectionId` but also, it really doesn't matter.